KristinJJ/GroupMovieRecommender
# Mg 27 fix styles scss pointer

Issue #27 - While reviewing PR #26, I encountered some stylings not coming through properly. after discussing with Kristin, she was able to determine that the angular.json was still pointing to "styles.css" instead of "styles.scss".

To test:
[ ] Verify angular.json styles element (line 34) has the ".scss" file ending,
[ ] npm start, and verify the input form is visible with a gray background, and hint text is in blue.

![image](https://user-images.githubusercontent.com/49133101/125861771-98ac758b-2bca-4be6-a9f3-4b9135c7ee51.png)
